src/main/scala/za/co/absa/commons/lang/CollectionImplicits.scala

```diff
@@ -21,6 +21,7 @@ import scala.collection.mutable
 import scala.language.higherKinds
 import scala.reflect.ClassTag
 
+@deprecated("Use type-specific ...Extension instead")
```

Comment:
Shouldn't this be accompanied by the current version to singify which version it was deprecated from? (multiple occurrences)

Reply:
Here is the thing - what version is the new version gonna be? I've heard we do semantic versioning in this library, but as far as I see, we pretty much always bump patch version (x.x.patch). So - is it gonna be 1.0.8, or 1.1.0?

Reply:
**1.1.0**

SemVer 2.0 spec makes it clear:
> Bug fixes not affecting the API increment the patch version, backwards compatible API additions/changes increment the minor version, and backwards incompatible API changes increment the major version.

Reply:
Yeah I know, I was just confused because so far even on new features patch version was bumped, not minor (which should be).

Reply:
Yeah, the automated release pipeline always increment patch version. So unless it's not explicitly intercepted on the PR level it's out there automatically. We need to be more careful about it. :+1:

Reply:
Changed version to 1.1.0-SNAPSHOT in the pom.xml. Is there anything else that needs to be done to make the plugin working as expected for this PR?